format/format.go

```diff
@@ -557,7 +557,7 @@ func (f *fumpter) applyPre(c *astutil.Cursor) {
 		f.stmts(node.Body)
 
 	case *ast.FieldList:
-		if node.NumFields() == 0 && f.inlineComment(node.Pos()) == nil {
+		if node.NumFields() == 0 && f.inlineComment(node.Pos()) == nil && len(f.commentsBetween(node.Pos(), node.End())) == 0 {
```

Comment:
```suggestion
		if node.NumFields() == 0 && len(f.commentsBetween(node.Pos(), node.End())) == 0 {
```

I think the `inlineComment` check is now redundant, as it's a subset of `commentsBetween` given that we check between the start and end of the entire field list.